### PR TITLE
Remove prefix underscores

### LINKED ISSUE #16278: Remove underscore prefix from property names
From [PSR-12](https://www.php-fig.org/psr/psr-12/#43-properties-and-constants):
> Property names MUST NOT be prefixed with a single underscore to indicate protected or private visibility. That is, an underscore prefix explicitly has no meaning.

Example:
```diff
-     private $_user;
+     private $user;
```

Files detected by phpcs: `vendor/bin/phpcs --warning-severity=4`

FILE | WARNINGS
-----|---------
libraries/classes/Bookmark.php | 5
libraries/classes/Console.php | 2
libraries/classes/DatabaseInterface.php | 9
libraries/classes/Encoding.php | 4
libraries/classes/File.php | 10
libraries/classes/Footer.php | 4
libraries/classes/Header.php | 12
libraries/classes/Index.php | 13
libraries/classes/IndexColumn.php | 6
libraries/classes/LanguageManager.php | 6
libraries/classes/Menu.php | 2
libraries/classes/OutputBuffering.php | 4
libraries/classes/RecentFavoriteTable.php | 3
libraries/classes/Response.php | 8
libraries/classes/SavedSearches.php | 6
libraries/classes/Scripts.php | 2
libraries/classes/Table.php | 3
libraries/classes/Theme.php | 1
libraries/classes/ThemeManager.php | 2
libraries/classes/Tracker.php | 1
libraries/classes/TwoFactor.php | 3
libraries/classes/Types.php | 1
libraries/classes/Config/ConfigFile.php | 9
libraries/classes/Config/Form.php | 2
libraries/classes/Config/FormDisplay.php | 9
libraries/classes/Config/PageSettings.php | 4
libraries/classes/Config/Forms/BaseFormList.php | 1
libraries/classes/Controllers/Table/FindReplaceController.php | 3
libraries/classes/Controllers/Table/SearchController.php | 7
libraries/classes/Controllers/Table/StructureController.php | 1
libraries/classes/Controllers/Table/ZoomSearchController.php | 7
libraries/classes/Database/Qbe.php | 27
libraries/classes/Gis/GisGeometryCollection.php | 1
libraries/classes/Gis/GisLineString.php | 1
libraries/classes/Gis/GisMultiLineString.php | 1
libraries/classes/Gis/GisMultiPoint.php | 1
libraries/classes/Gis/GisMultiPolygon.php | 1
libraries/classes/Gis/GisPoint.php | 1
libraries/classes/Gis/GisPolygon.php | 1
libraries/classes/Gis/GisVisualization.php | 4
libraries/classes/Plugins/TwoFactorPlugin.php | 3
libraries/classes/Plugins/Auth/AuthenticationCookie.php | 2
libraries/classes/Plugins/Export/ExportCodegen.php | 1
libraries/classes/Plugins/Export/ExportPdf.php | 2
libraries/classes/Plugins/Export/ExportSql.php | 1
libraries/classes/Plugins/Export/ExportXml.php | 2
libraries/classes/Plugins/Import/ImportCsv.php | 1
libraries/classes/Plugins/Import/ImportMediawiki.php | 1
libraries/classes/Plugins/Schema/Dia/DiaRelationSchema.php | 6
libraries/classes/Plugins/Schema/Eps/EpsRelationSchema.php | 3
libraries/classes/Plugins/Schema/Pdf/Pdf.php | 7
libraries/classes/Plugins/Schema/Pdf/PdfRelationSchema.php | 15
libraries/classes/Plugins/Schema/Pdf/TableStatsPdf.php | 1
libraries/classes/Plugins/Schema/Svg/SvgRelationSchema.php | 7
libraries/classes/Plugins/TwoFactor/Application.php | 1
libraries/classes/Properties/Options/OptionsPropertyGroup.php | 1
libraries/classes/Properties/Options/OptionsPropertyItem.php | 3
libraries/classes/Properties/Options/OptionsPropertyOneItem.php | 5
libraries/classes/Properties/Options/Groups/OptionsPropertySubgroup.php | 1
libraries/classes/Properties/Plugins/ExportPluginProperties.php | 1
libraries/classes/Properties/Plugins/PluginPropertyItem.php | 5
libraries/classes/Server/SysInfo/WindowsNt.php | 1
test/classes/DatabaseInterfaceTest.php | 1
test/classes/IndexTest.php | 1
test/classes/VersionInformationTest.php | 1
test/classes/Controllers/Table/RelationControllerTest.php | 1
test/classes/Controllers/Table/SearchControllerTest.php | 1
test/classes/Controllers/Table/StructureControllerTest.php | 1
test/classes/Stubs/DbiDummy.php | 1
test/classes/Stubs/Response.php | 2
test/selenium/CreateRemoveUserTest.php | 2

### PR DESCRIPTION
**Pull request by Vahalaru**
### Description

Please describe your pull request.
I have removed the prefix underscore throughout the files

Fixes #16278 

Before submitting pull request, please review the following checklist:

- [x ] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x ] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x ] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x ] Every commit has a descriptive commit message.
- [x ] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [ x] Any new functionality is covered by tests.

### CODE REVIEW

File: libraries/classes/DatabaseInterface.php [hunk 1]
```diff
@@ -383,7 +383,7 @@ public function getTablesFull(
             );
 
             // for PMA bc:
-            // `SCHEMA_FIELD_NAME` AS `SHOW_TABLE_STATUS_FIELD_NAME`
+            // `SCHEMA_FIELD_NAME` AS `SHOWtable_STATUS_FIELD_NAME`
```
Comment (williamdes) [inline libraries/classes/DatabaseInterface.php:386]:
```suggestion
            // `SCHEMA_FIELD_NAME` AS `SHOW_TABLE_STATUS_FIELD_NAME`
```

File: libraries/classes/DatabaseInterface.php [hunk 2]
```diff
@@ -541,14 +541,14 @@ static function ($a, $b) {
                 }
 
                 if ($limit_count) {
-                    $each_tables = array_slice(
-                        $each_tables,
+                    $eachtables = array_slice(
```
Comment (williamdes) [inline libraries/classes/DatabaseInterface.php:544]: Please use camel case :)

File: libraries/classes/DatabaseInterface.php [hunk 3]
```diff
@@ -708,8 +708,8 @@ public function getDatabasesFull(
                 }
 
                 while ($row = $this->fetchAssoc($res)) {
-                    $databases[$database_name]['SCHEMA_TABLES']++;
-                    $databases[$database_name]['SCHEMA_TABLE_ROWS']
+                    $databases[$database_name]['SCHEMAtableS']++;
+                    $databases[$database_name]['SCHEMAtable_ROWS']
```
Comment (williamdes) [inline libraries/classes/DatabaseInterface.php:712]:
```suggestion
                    $databases[$database_name]['SCHEMA_TABLES']++;
                    $databases[$database_name]['SCHEMA_TABLE_ROWS']
```

File: libraries/classes/DatabaseInterface.php (hunk 2 above)
Comment (Vahalaru) [reply libraries/classes/DatabaseInterface.php:544]: Updated, how do I update my pull request?

File: libraries/classes/DatabaseInterface.php (hunk 3 above)
Comment (Vahalaru) [reply libraries/classes/DatabaseInterface.php:712]: Updated, how do I update my pull request?

File: libraries/classes/DatabaseInterface.php (hunk 1 above)
Comment (Vahalaru) [reply libraries/classes/DatabaseInterface.php:386]: Updated, how do I update my pull request?

**Comment by Vahalaru** [comment]
This pull request had a lot of mistakes so I made a new one o couldn't figure out how to update this one

**Comment by williamdes** [comment]
Duplicate of #16293

**Pull request closed without merging**